src/WorkerChannel.ts

```diff
@@ -154,4 +194,54 @@ export class WorkerChannel {
       resultCallback(err);
     }
   }
+
+  /**
+   * Worker sends the host information identifying itself
+   */ 
+  public startStream(requestId: string, msg: rpc.StartStream): void {
+    // Not yet implemented
+  }
+
+  /**
+   * Message is empty by design - Will add more fields in future if needed
+   */ 
+  public workerHeartbeat(requestId: string, msg: rpc.WorkerHeartbeat): void {
+    // Not yet implemented
+  }
+
+  /**
+   * Warning before killing the process after grace_period
+   * Worker self terminates ..no response on this
+   */ 
+  public workerTerminate(requestId: string, msg: rpc.WorkerTerminate): void {
+    // Not yet implemented
+  }
+
+  /**
+   * NOT USED
+   */ 
+  public workerStatusRequest(requestId: string, msg: rpc.WorkerStatusRequest): void {
+    // Not yet implemented
+  }
+
+  /**
+   * Host notifies worker of file content change
+   */   
+  public fileChangeEventRequest(requestId: string, msg: rpc.FileChangeEventRequest): void {
+    // Not yet implemented
+  }
+
+  /**
+   * Host requests worker to cancel invocation
+   */ 
+  public invocationCancel(requestId: string, msg: rpc.InvocationCancel): void {
+    // Not yet implemented
+  }
+  
+  /**
+   * Environment variables from the current process
+   */ 
+  public functionEnvironmentReloadRequest(requestId: string, msg: rpc.IFunctionEnvironmentReloadRequest): void {

```

Comment:
Please implement functionEnvironmentReloadRequest - This method needs to reload environment variables on the node process